alert-policies/java/TransactionErrors.yml

```diff
@@ -1,14 +1,12 @@
 name: Transaction Errors
 
 description: |+
-  This alert is triggered when the the transactions fail more than 10% of the time in 5 minutes.
+  This alert is triggered when the transaction errors are higher than 5 for 5 minutes.
```

Comment:
Could we say something like this instead: 
`This alert is triggered when the transaction error count is higher than 5 for 5 minutes.